bpf/lib/lb.h

```diff
@@ -1695,12 +1702,13 @@ static __always_inline int lb4_local(const void *map, struct __ctx_buff *ctx,
  */
 static __always_inline void lb4_ctx_store_state(struct __ctx_buff *ctx,
 						const struct ct_state *state,
-					       __u16 proxy_port)
+					       __u16 proxy_port, __u8 cluster_id)
 {
 	ctx_store_meta(ctx, CB_PROXY_MAGIC, (__u32)proxy_port << 16);
 	ctx_store_meta(ctx, CB_BACKEND_ID, state->backend_id);
 	ctx_store_meta(ctx, CB_CT_STATE, (__u32)state->rev_nat_index << 16 |
 		       state->loopback);
+	ctx_store_meta(ctx, CB_CLUSTER_ID, (__u32)cluster_id << 24);
```

Comment:
Why do we need to do the left-shift? Can't we just cast to u32? Same argumentation applies to the stores above.